avm/res/app-configuration/configuration-store/main.bicep

```
@@ -26,10 +26,10 @@ param sku string = 'Standard'
 param createMode string = 'Default'
 
 @description('Optional. Disables all authentication methods other than AAD authentication.')
-param disableLocalAuth bool = false
+param disableLocalAuth bool = true
 
-@description('Optional. Property specifying whether protection against purge is enabled for this configuration store.')
-param enablePurgeProtection bool = false
+@description('Optional. Property specifying whether protection against purge is enabled for this configuration store.Defaults to true unless sku is set to Free, since purge protection is not available in Free tier.')
```

Sorry, just noticed this minor one
```suggestion
@description('Optional. Property specifying whether protection against purge is enabled for this configuration store. Defaults to true unless sku is set to Free, since purge protection is not available in Free tier.')
```

---

Thanks @eriqua I have committed this change.

---

Hey @JFolberth thanks. We are very close! Please run the Set-AVMModule utility to reflect the change in the main.json as well. Also please rerun the pipeline so that it can hopefully have a green badge